third_party/amd/include/TritonAMDGPUTransforms/Passes.h

```diff
@@ -1,6 +1,7 @@
 #ifndef TRITON_THIRD_PARTY_AMD_INCLUDE_TRITONAMDGPUTRANSFORMS_PASSES_H_
 #define TRITON_THIRD_PARTY_AMD_INCLUDE_TRITONAMDGPUTRANSFORMS_PASSES_H_
 
+#include "mlir/Dialect/LLVMIR/ROCDLDialect.h"
```

Comment:
This header include is not needed?

Reply:
No, deleted.

Reply:
Oh, it's required.
I've set dependency to the rocdl dialect on pingpong pass in Passes.td. tablegen inserted registry.insert<mlir::ROCDL::ROCDLDialect>() in the Passes.h.inc this requires header.